Assignments/Program.cs

```diff
@@ -1 +1,131 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T2 - NUMBER TO WORDS AND ROMAN NUMERALS CONVERTER 
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T2 {
+   public static void Main () {
+      List<int> wTestCases = new () { 0, 1, 4, 9, 10, 11, 16, 25, 36, 49, 50, 81, 100, 109, 125, 216,
+                                    343, 729, 1000, 12001, 66000, 123450, 1000909, 987654321, 100000001 },
+                rTestCases = new () { 0, 1, 4, 5, 9, 10, 47, 94, 100, 345, 400, 499, 500, 900, 998, 1000, 
+                                    1500, 1729, 1888, 3495, 4000, 3999 };
+      Console.WriteLine ("\x1B[4m" + "Numbers to Words:" + "\x1B[0m");
+      PrintResult (wTestCases, true);
+      Console.WriteLine ("\n\x1B[4m" + "Numbers to Romans:" + "\x1B[0m");
+      PrintResult (rTestCases, false);
+   }
+
+   /// <summary>Prints the Words or Roman numerals equivalent to the input number</summary>
+   public static void PrintResult (List<int> input, bool toWords) {
+      for (int i = 0; i < input.Count; i++) {
+         int n = input[i];
+         Console.Write ($"{i + 1,2}) {input[i],10} ==> ");
+         if (!toWords && n > 3999) {
+            Console.WriteLine ($"Invalid Input!");

```

Comment:
```suggestion
            Console.WriteLine ($"Invalid input!");

```
User messages and comments should be in proper English and grammar